packages/helix-shared-config/test/sitemapconfigs.test.js

```diff
@@ -59,10 +76,11 @@ describe('Sitemap Config Loading', () => {
         }
       } catch (e) {
         if (test.error) {
-          assert.equal(e.toString(), test.error);
-        } else {
-          throw e;
+          if (e.toString().startsWith(test.error)) {
+            return;
+          }
```

Comment:
error messages are quite long, just ensure the initial description in `error` matches